highfidelity/hifi
# Avatar without Neck Joint bug 14620

This pr addresses the bug https://highfidelity.fogbugz.com/f/cases/14620/Avatar-without-neck-joint-renders-incorrectly-from-first-person-view-but-fine-in-3rd

The fix is to use the parent of the head joint as the proxy neck.  Then the cauterized model will render correctly and solve the bug in this case.

### Test Plan
Unusual rendering behavior with avatar which is missing a neck bone.  It renders great in 3rd person view and to other users in the same domain.  

1. Load and wear this avatar http://hifi-content.s3.amazonaws.com/caitlyn/scratch/nekoMMD/nekoMMD.fst
2. Observe how she looks in 3rd person view.  Notice she looks fine.
3. Switch to first person camera view. Notice you should now see yourself correctly
4. Look up into the sky at hifi://portalarium
5. You should no longer see the avatar mesh is projected very, very high up above, like a shadow against the skybox:


Look at the screenshot, that blue area is the model from below, crotch area
![image](https://user-images.githubusercontent.com/22531620/43926858-65f63ea4-9be0-11e8-97bf-39b2707ea1d6.png)
